src/nvim/testdir/test_python3.vim

```diff
@@ -2,7 +2,7 @@
 " TODO: move tests from test88.in here.
 
 if !has('python3')
-  finish
+  throw 'Skipped: does not have +python3'
```

Comment:
`+python{2,3}` never exists. The feature is called `python{2,3}` (without `+`). Full list of features is at `:h feature-list` (well, it's missing "ruby"...).

Reply:
Yes, that's only for naming, but I agree that is confusing - only temporary though.

I do not get why it is not available during tests, but before - i.e. the tests gets skipped, but I'd expect them to run (and fail).